ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -276,28 +276,6 @@ wait "$!"
 
 # Platform specific manifests adjustments
 case "${CLUSTER_TYPE}" in
-aws|aws-arm64|aws-usgov)
-	if [[ "${SPOT_MACHINES}" == 'true' ]]; then
```

Comment:
I guess if this were `${SPOT_MACHINES:-}` it wouldn't have broken

Reply:
Yup that's probably all it needs. SOP here would be unrevert this revert, layer a new commit with that change and test.

Reply:
That's fine, we can do that once this is merged